checkov/cloudformation/checks/resource/aws/SecretManagerSecretEncrypted.py

```diff
@@ -20,5 +22,8 @@ def scan_resource_conf(self, conf: DictNode) -> CheckResult:
                     return CheckResult.PASSED
         return CheckResult.FAILED
 
+    def get_evaluated_keys(self) -> List[str]:
+        return ['Properties/KmsKeyId']
```

Comment:
also `Properties/KmsKeyId/aws`

Reply:
@YaaraVerner The checks validates `kms_key_id` doesn't contain the string `aws/`, it isn't another key in the template.